android/fastlane/actions/get_environment_info.rb

```diff
@@ -0,0 +1,42 @@
+require 'dotenv'
```

Comment:
All this action is the same for iOS Fastlane and it would be great to abstract this just once. Think a way to do that and then send me an email to walterthemostlittlecapo@gmail.com to validate it with me

Reply:
mm nice to have jaja keeping them separated is easier for people that only needs ios or android version